libheif/heif_cxx.h

```diff
@@ -549,6 +559,22 @@ namespace heif {
     return heif_image_handle_has_alpha_channel(m_image_handle.get()) != 0;
   }
 
+  inline int ImageHandle::get_luma_bits_per_pixel() const noexcept {
+    return heif_image_handle_get_luma_bits_per_pixel(m_image_handle.get()) != 0;
```

Comment:
Why the "!= 0" ? That does not seem correct. Same for the other functions in this block.